src/Form/Data/TableColumn.php

```diff
@@ -16,7 +16,8 @@ final class TableColumn
     private ?string $iconProperty = null;
     private ?bool $dateTimeProperty = null;
     private ?bool $dataLinks = null;
-    private ?bool $arrayOfStrings = null;
+    private string $class = 'nowrap';
```

Comment:
Hard dependency on the styling system that is used.. Maybe use constants and translate them inside the main template of twig?
```suggestion
    private string $class = 'nowrap';
```

Reply:
Agree see my coming PR